autogen_stubs.sh

```diff
@@ -22,12 +22,43 @@ fixup() {
   grep FIXME_STUB $1 || true
 }
 
+generate_opencl_error_codes(){
```

Comment:
no need to git clone, do the same way as it's done in `generate_opencl`. I think you can add just `cl_ext.h` there and parse errors. Also errors are negative values, can we just build a dict Dict[error int -> str] in ops_gpu with a one-liner?

Reply:
are the other errors not important so i can just get what is present on cl_ext.h.
There's errors on cl.h, cl_egl.h, cl_dx9_media_sharing.h, cl_d3d11.h and cl_d3d10.h files.

About the one-liner, it's possible , it'll just not be genned by the bash script

Reply:
"do the same way as it's done in generate_opencl", you mean, just assume the header file it's already available on the local directory?, I don't get how that's better than git clone if the dir doesn't exist. OpenCL headers are not default to be present on all linux distros.

Reply:
cl_ext only have the -1141 error code, cl.h have 72 error codes

Reply:
just a note, there's two error codes that are not negative values, they are hexadecimal [0x40D4] [0x40D5]

Reply:
@nimlgen can you help me on the questions above? Any other thing missing for the PR merge?

Reply:
@nimlgen i think i got your idea, did a simplification of the seds